packages/renderer/src/lib/troubleshooting/TroubleshootingGatherLogs.svelte

```diff
@@ -0,0 +1,56 @@
+<script lang="ts">
+import { faFileLines, faScroll } from '@fortawesome/free-solid-svg-icons';
+import Button from '../ui/Button.svelte';
+import Fa from 'svelte-fa';
+import { generateLogFileName } from '../../../../main/src/plugin/troubleshooting';
+
+let logs: string[] = [];
+
+// Save files as a zip file (we first ask the user for the dialog, and then save the files to the filepath)
+async function saveLogsAsZip() {
+  const result = await window.saveFileDialog('Save Logs as .zip', generateLogFileName('podman-desktop') + '.zip');
+  if (!result.canceled && result.filePath) {
+    const logFiles = await window.saveTroubleshootingLogs(result.filePath);
+    logs = logFiles;
+  }
+}
+</script>
+
+<div class="flex flex-col w-full m-4 bg-charcoal-600 p-4 rounded-lg">
+  <div class="flex flex-row align-middle items-center w-full">
+    <Fa size="30" class="pr-3 text-gray-700" icon="{faFileLines}" />
+    <div class="text-xl">Gather Log Files</div>
+    <div class="flex flex-1 justify-end"></div>
+  </div>
+  <div class="mt-4">Bundle all available logs into a .zip</div>
+  <div class="mt-4">
+    <Button
+      on:click="{() => {
+        saveLogsAsZip();
+      }}"
+      title="Collect logs for must gather tool"
+      icon="{faScroll}">Collect and save logs as .zip</Button>
+  </div>
+  {#if logs.length > 0}
+    <div class="h-full overflow-auto p-2 bg-charcoal-800 mt-3">
+      <ul aria-label="logs">
+        {#each logs as log}
+          <li>
+            <div class="flex flex-row align-middle items-center">
+              <div class="font-mono text-[10px] font-thin">
+                {log}.txt generated
```

Comment:
Assumes that all files end in .txt and are generated. With the prior comment the correct filename would be here so you can drop the .txt, and I'd drop 'generated' too, since we don't know if the file was just copied in or generated. i.e. just use `{log}`.